autogen/agentchat/conversable_agent.py

```diff
@@ -637,16 +652,29 @@ def send(
         Raises:
             ValueError: if the message can't be converted into a valid ChatCompletion message.
         """
+        action_event = (
```

Comment:
Can you take a look at runtime_logging module and work with @lalo to add new events through the logging API rather than adding agent ops specific code here? 

Can you also implement an agent ops logger to cover those new events. You can take a look at the ongoing PR by our colleague at Microsoft's cosmos db as a reference. https://github.com/microsoft/autogen/pull/2329

We will ask you to maintain the code once issues arise.

Reply:
The goal is to have a custom logger that user can pass into runtime_logging, and the custom logger (I.e, agent ops logger) can receive all events logged by the runtime_logging.